### DISCUSSION TITLE
Create executable for all platforms

### DISCUSSION THREAD

**Discussion by ithilbor** [tauri-apps/tauri#4057] [Q&A] [answered]
Hi, i'm on mac with Tauri and when i lunch the bundle (yarn tauri build --verbose) the only executable that create is for mac, someone that know how to tell Tauri to build also the executables for windows and Linux?

**Comment by FabianLars** [comment, marked as answer]
We don't support cross-platform compilation yet (hopefully "yet"). Until then you have to resolve to tools like GitHub action or local VMs.

**Reply by ithilbor** [reply]
So if i'm on Mac in order to build my application for windows and Linux i Just need to set up a VM with that OS lunch the build inside of it and it will compile it for that OS?

**Reply by FabianLars** [reply]
Yes. You have to do the same you do on macOS to get a macOS app, inside a linux and windows VM to get their respective bundles. (like, install the dependencies, clone your project, run `tauri build`)

**Reply by ithilbor** [reply]
Super clear, thank you so much, i really like Tauri, for now it seams a lot better of Electron it has a very big potential i hope it will continue scaling by adding new features in future and maybe one of them it will be the support for cross-platform, so exited to see that.
Maybe if i will have more time i'll check to see if a contribute to this project, thanks again and have a nice day!

**Reply by ithilbor** [reply]
PS: it' important also the arch for the build?

**Reply by FabianLars** [reply]
Well preferably your VM is x86_64, but if it can only be arm based that should fine for windows, but probably causes problems on linux (arm to x86 cross compiling is pretty rare, so i don't have much info on that one).
